src/cmd_line/commands/file.ts

```diff
@@ -16,6 +16,7 @@ export interface IFileCommandArguments extends node.ICommandArgs {
   bang?: boolean;
   position?: FilePosition;
   lineNumber?: number;
+  dontCreateFile?: boolean;
```

Comment:
can you do the negation and rename something more descriptive like `createFileIfNotExists`

Reply:
Thanks so much for this PR! <3